cmd/cmd_pkg.go

```diff
@@ -290,6 +301,61 @@ func cmdPkgLogin(cmd *cobra.Command, args []string) {
 	fmt.Printf("Login Successful as user %s\n", resp.Username)
 }
 
+func cmdPkgPush(cmd *cobra.Command, args []string) {
+	packageName := args[0]
+	creds, err := api.ReadCredsFromLocal()
+	if err != nil {
+		if err == api.ErrCredentialsNotExist {
+			// for a better error message
+			log.Fatal(errors.New("user is not logged in. use 'ops pkg login' first"))
+		} else {
+			log.Fatal(err)
+		}
+	}
+	pkgList, err := api.GetLocalPackageList()
+	if err != nil {
+		log.Fatal(err)
+	}
+	localPackages := filepath.Join(api.GetOpsHome(), "local_packages")
+	packageFound := false
+	var foundPkg api.Package
+	for name, pkg := range *pkgList {
+		if name == packageName {
+			packageFound = true
+			foundPkg = pkg
+			break
+		}
+	}
+	if !packageFound {
```

Comment:
instead of including this bool you could just look for a required package field like the sha

Reply:
Understood

Reply:
done